invenio_vocabularies/resources/serializer.py

```diff
@@ -45,6 +45,6 @@ class VocabularyL10NListSchema(Schema):
 
     def get_hits(self, obj_list):
         """Apply hits transformation."""
-        schema = self.context["schema_cls"]()
+        schema = self.context["object_schema_cls"]()
```

Comment:
Shouldn't all these schemas inherit from the base schema in flask-resources?

Reply:
I guess we could, we would also dump the non-existing aggregations but I don't think that's a problem